content/collections/source-catalog/en/ga4.md

```diff
@@ -0,0 +1,150 @@
+* [Create a Service Account](https://cloud.google.com/iam/docs/service-accounts-create#console) with permissions granted for the bucket and tables you want to ingest, then [get the service account key](https://cloud.google.com/iam/docs/keys-create-delete). The Service Account must have the following roles granted to it:
+  * **BigQuery**
+    * BigQuery Job User at the project level.
+    * BigQuery Data Viewer at the resource level necessary to access your data to be ingested. If your data is in a single table, just grant the service account BigQuery Data Viewer for that table resource. If the query requires multiple tables or datasets, grant BigQuery Data Viewer on all tables or datasets in the query.
+  * **Cloud Storage**
+    * Storage Admin on the GCS bucket you're using for ingestion.
+* Depending on your company's network policy, you may need to add the following IP addresses to your allowlist to allow Amplitude's servers to access your BigQuery instance:
+  * Amplitude US IP addresses:
+    * 52.33.3.219
+    * 35.162.216.242
+    * 52.27.10.221
+  * Amplitude EU IP addresses:
+    * 3.124.22.25
+    * 18.157.59.125
+    * 18.192.47.195
+
+## How Amplitude imports your GA4 data
+
+Here’s a step-by-step breakdown of how Amplitude imports your GA4 data:
+
+1. Export GA4 Data to BigQuery
+
+    First, your GA4 data is exported to your BigQuery project through BigQuery Linking. This process sets up an automated flow of event data from GA4 to BigQuery.
+
+2. Unload Data from BigQuery to Your GCS Bucket
+
+    To handle BigQuery’s export limitations (e.g., export size restrictions), we unload your data from BigQuery to the GCS bucket you provided. This ensures data readiness for ingestion into Amplitude.
+
+    * Delay Handling:
+
+      To ensure your data is complete, we account for [delays caused by GA4’s export process](https://support.google.com/analytics/answer/9358801?hl=en):
+      * 72 hours for daily exports.
+      * 24 hours for streaming exports.
+
+    * Catch-up Importing:
+
+        If there’s a backlog, we use a catch-up strategy to process data in batches of 10 tables per day. For example:
+
+        * If you configure your import to start from the event table on 2025-01-01, and today’s date is 2025-01-24, we import:
+          * 2025-01-01 to 2025-01-10 on 2025-01-24.
+          * 2025-01-11 to 2025-01-20 on 2025-01-25.
+          * 2025-01-21 to 2025-01-23 on 2025-01-26.
+          * 2025-01-24 on 2025-01-27 (due to the 72-hour delay for daily exports).
+
+    During the unload process, we also transform your data into the format required by Amplitude for seamless ingestion.
+
+3. Import Data from GCS to Amplitude
+
+    Once your data is synced to the specified GCS bucket, it will be imported into Amplitude. Please note that it may take some time for your data to be fully available in Amplitude. We appreciate your patience during this process.
```

Comment:
let's rephrase it by removing the call out in waiting since that could raise some concerns. Something like

```
Once data is ready in GCS bucket, we will import data into Amplitude on your behalf.
```